hardening/kickstart/main.fmf

```diff
@@ -61,8 +59,7 @@ adjust:
     environment+:
         WITH_FIPS: 1
     adjust+:
-      - when: distro >= rhel-10
-        enabled: false
+      - enabled: false
```

Comment:
All of these had (weirdly) `distro >= rhel-10` because they were meant to be copy/pastes of per-profile `.fmf` defines used by other tests.

I guess the idea was that we could `diff` all `.fmf` files with per-profile configurations and "backport" any changes (like `WITH_FIPS` to all of them). But that diffing never manifested, so I guess it makes sense to *not* maintain the copy/paste nature of these.

We should probably discuss this more in detail - ie. we can go one step further and remove always-`false` sections, which we currently only have due to the `because` explanation.